src/shared/components/header/header.component.html

```diff
@@ -3,13 +3,20 @@
 
   <button
     cdkOverlayOrigin
-    class="btn btn-add"
+    class="btn"
     (click)="openDialog()">
     {{ buildTranslationKey('addButtonText') | translate }}
   </button>
 </div>
 
 <div class="search-sort">
-  <app-search></app-search>
+  <app-search [placeholder]="buildTranslationKey('searchPlaceholder') | translate"></app-search>
+
+  <app-dropdown
+    [formControl]="sortControl"
+    [icon]="iconSrc"
+    [options]="options"
+    [name]="options[0].value">
```

Comment:
it's better to pass [selectedOption]="options[0]" and then you can do manipulations of settings values inside the dropdown component